pyke/targetpixelfile.py

```diff
@@ -56,23 +61,38 @@ class KeplerTargetPixelFile(TargetPixelFile):
     """
 
     def __init__(self, path, aperture_mask=None,
-                 quality_bitmask=KeplerQualityFlags.DEFAULT_BITMASK,
+                 quality_mask='default',quality_bitmask=None,
                  **kwargs):
         self.path = path
         self.hdu = fits.open(self.path, **kwargs)
-        self.quality_bitmask = quality_bitmask
-        self.quality_mask = self._quality_mask(quality_bitmask)
+        self.quality_mask = self._quality_mask(quality_bitmask,quality_mask)
         self.aperture_mask = aperture_mask
 
-    def _quality_mask(self, quality_bitmask):
+    def _quality_mask(self, bitmask, mask):
         """Returns a boolean mask which flags all good-quality cadences.
 
         Parameters
         ----------
         quality_bitmask : int
             Bitmask. See ref. [1], table 2-3.
         """
-        return (self.hdu[1].data['QUALITY'] & quality_bitmask) == 0
+        if bitmask == None:
+            if (mask is None) or (mask is 'None'):
+                bitmask=None
+            if (mask is 'default'):
+                bitmask=KeplerQualityFlags.DEFAULT_BITMASK
+            if (mask is 'conservative'):
+                bitmask=KeplerQualityFlags.CONSERVATIVE_BITMASK
+            if (mask is 'hard'):
+                bitmask=KeplerQualityFlags.QUALITY_ZERO_BITMASK
+            if not (mask in [None,'None','default','conservative','hard']):
+                bitmask=KeplerQualityFlags.DEFAULT_BITMASK
+            self.quality_bitmask=bitmask
+        if bitmask is None:
+            return ~np.zeros(len(self.hdu[1].data['TIME']),dtype=bool)
+        else:
+            return (self.hdu[1].data['QUALITY'] & bitmask) == 0
+
```

Comment:
This function is exactly the same in ``KeplerLightCurveFile`` class.
Maybe we would want to move it to ``utils.py`` as a standalone function?